madmom/ml/nn/layers.py

```diff
@@ -44,16 +51,24 @@ def activate(self, data):
         """
         raise NotImplementedError('must be implemented by subclass.')
 
+    @staticmethod
```

Comment:
Yes, this was a normal method before, changed it because some checker criticised it. Will change it back.